src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(a => a.sex === 'm');
+
+  if (century) {
+    men = men.filter(a => {
+      return Math.ceil(a.died / 100) === century;
+    });
+  }
```

Comment:
```suggestion
  const men = people.filter(person =>
    century
      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
      : person.sex === 'm');
```
В остальных кейсах можно сделать так же